template/deployments/helm/charts/templates/_helpers.tpl

```diff
@@ -1,51 +1,34 @@
 {{/*
 Expand the name of the chart.
 */}}
-{{- define "{{APPNAME}}.name" -}}
+{{- define "helper.name" -}}
```

Comment:
hardcoding to the top level to helper doesn't seem to help much here

> Since templates in subcharts are compiled together with top-level templates, it is best to name your templates with chart specific names. A popular naming convention is to prefix each defined template with the name of the chart: {{ define "mychart.labels" }}.


what is the point of these changes? I don't understand the context of why we are switching from the bestpractice for charts to this

Reply:
the idea is to migrate all the templates to go templates, so it helps in version control.